### ISSUE TITLE
Docker Entrypoint is a Command

### ISSUE THREAD

**Issue by iMerica** [PostHog/posthog#201]
## Description
Gunicorn is running as an entrypoint instead of as a command. This prevents composability of the docker image and makes Posthog less configurable.

### Steps to Reproduce
Lets say you want to run Posthog locally on port 8001 using Compose. The straightforward way would be to do 

```yaml
  posthog:
    image: posthog/posthog:latest
    command: python manage.py runserver "0.0.0.0:8001"
    environment:
       // fields
    ports:
      - "8001:8001"
```

Except this would not work because the entrypoint never terminates or forwards `@$`. 

## Expected behavior
I expect the docker image to be more flexible so that I can use the same image to run locally, run migrations, run in production, run one off django commands etc.

## Solution
Replace `ENTRYPOINT ["./bin/docker"]` with  `CMD ["./bin/docker"]`. Optionally add a new ENTRYPOINT that does something like test a db connection before continuing.

**Comment by iMerica** [comment]
I was able to just override the entrypoint with `[]` to disable it. 

## Workaround

Run PostHog locally:
```yaml
  posthog: &posthog
    image: posthog/posthog:latest
    command: python manage.py runserver "0.0.0.0:8001"
    entrypoint: []
    environment:
      POSTHOG_SECRET_KEY: 'XXXXXXXXX'
      DATABASE_URL: "XXXXXX"
      IS_DOCKER: "true"
      DEBUG: 1
    ports:
      - "8001:8001"
    depends_on:
      - postgres
  posthog-migrations:
    <<: *posthog
    command: python manage.py migrate
    ports: []

```

`docker-compose up -d`

**Comment by timgl** [comment]
@iMerica Thanks! That's fixed now :-). I've released `1.0.1` with the fix.

**Comment by timgl** [comment]
@iMerica Also, I'd recommend using Gunicorn rather than the Django development server in production: `gunicorn posthog.wsgi --config gunicorn.config.py --bind 0.0.0.0:8001 --log-file -`
